src/script/ceph-release-notes

```diff
@@ -110,6 +110,31 @@ def split_component(title, gh, number):
         else:
             return 'UNKNOWN: ' + title
 
+def _title_message(commit, pr, strict):
+    title = pr['title']
+    message_lines = commit.message.split('\n')
+    if strict or len(message_lines) < 1:
+        return (title, None)
+    lines = []
+    for line in message_lines[1:]:
+        reviewed=re.compile(r"Rev(.*)By", re.IGNORECASE)
```

Comment:
and would be better to move the `re.compile()` out of the function, so we don't need to compile a new `re` for every PR.